docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -948,6 +948,7 @@ Default templates for project boards:
 ## Git (`git`)
 
 - `PATH`: **""**: The path of Git executable. If empty, Gitea searches through the PATH environment.
+- `HOME_PATH`: **%(APP_DATA_PATH)/home**: The HOME directory for Git.
```

Comment:
I prefer to add more description here.

If you can confirm Gitea is the only application running in this environment, you can set it to `/home/git` if it running in UNIX-style OS as `git` user.

Reply:
This is where the caveat `that Gitea's git calls will use`. The authorized_keys is used by `sshd` not `git`.

Reply:
I think there are two request routes

1) sshd -> gitea serv/hook -> web request to gitea web -> invoke git command with $HOME
2) Git Smart HTTP -> invoke git command with $HOME

So `.ssh` should not be created. Only `.gitconfig` and `.gnupg` are possbile.

if we set git.HOME_PATH to /home/git, that means we fallback from standalone git config to a global git config.

Reply:
The .ssh is accident created in https://github.com/go-gitea/gitea/blob/c273dea5085aece451b10d78799d879c28e8a752/cmd/serv.go#L65 we can fix that in another PR.

Reply:
Just wrote one:

* https://github.com/go-gitea/gitea/pull/20289